src/client/pages/Landing/landingPageData.ts

```diff
@@ -29,13 +29,13 @@ export const productsData: ProductsData = {
   SE: [
     {
       id: 'swedishNew',
-      linkSlug: '/new',
+      linkSlug: '/home-accident-needer',
```

Comment:
I'm not sure we should change the routes though. I'm thinking in terms of tracking and if we links to the flows directly. But we can check with Sonny I guess

Reply:
Aha, very true!